dpi/src/lib.rs

```diff
@@ -1205,4 +1207,21 @@ mod tests {
         let _ = format!("{:?}", Size::Physical((1, 2).into()).clone());
         let _ = format!("{:?}", Position::Physical((1, 2).into()).clone());
     }
+
+    #[test]
+    fn ensure_copy_trait() {
+        fn is_copy<T: Copy>() {}
+
+        is_copy::<LogicalUnit<i32>>();
+        is_copy::<PhysicalUnit<f64>>();
+        is_copy::<PixelUnit>();
+
+        is_copy::<LogicalSize<i32>>();
+        is_copy::<PhysicalSize<f64>>();
+        is_copy::<Size>();
+
+        is_copy::<LogicalPosition<i32>>();
+        is_copy::<PhysicalPosition<f64>>();
+        is_copy::<Position>();
+    }
```

Comment:
Can you add a similar test for `PartialEq`?

Reply:
Done